options/options.go

```diff
@@ -105,6 +106,26 @@ type TerragruntOptions struct {
 	// packages can use the command without a direct reference back to the cli package (which would create a
 	// circular dependency).
 	RunTerragrunt func(*TerragruntOptions) error
+
+	Context TerragruntContext
```

Comment:
What's this?

Reply:
Because terragrunt is able to to include config files we must be able to store some information what config files has been included and what's even more important from where they have been included because to fully support late interpolations we must save the include path information because the interpolation functions `path_relative_to_include` and `path_relative_from_include`rely on them.